### ISSUE TITLE
Optimize loading state behavior for assistive technologies

### ISSUE THREAD

**Issue by o-hook** [porsche-design-system/porsche-design-system#2495]
Scope
---
The PDS has several components which can have an "intermediate" `loading` state (while in focus). Those components must expose this state to assistive technology users (e.g. screen reader) including telling the user when loading starts and when it ends.

Out of Scope
---
- No internationalisation, predefined English texts only

Notes  
---
- See open Polaris issue as inspiration: https://github.com/Shopify/polaris/issues/1212
- Article about an accessible microinteraction button: https://accessabilly.com/an-accessible-microinteractions-button/
- Article about multi-functional buttons: https://adrianroselli.com/2021/01/multi-function-button.html
- Handling of the  begin/end loading state should be equal for those components

Acceptance criteria
---
- [ ] All components with loading states should expose begin/end of this state to assistive technology users

Subtasks
---
- [ ] Task
